### ISSUE TITLE
Paper metadata

### ISSUE THREAD

**Issue by arokem** [openjournals/joss#298]
In the future, when we are doing analyses of the kind that went into the first-year summary, it might be useful for us to have some metadata that is currently hard to find: not only the language of implementation of the software, but also things like the research domain to which it contributes, etc. 

Is there an ontology of research software we could use and ask people to select from?

**Comment by arfon** [comment]
> In the future, when we are doing analyses of the kind that went into the first-year summary, it might be useful for us to have some metadata that is currently hard to find: not only the language of implementation of the software, but also things like the research domain to which it contributes, etc.

Yeah, that would be really interesting. Detecting programming language should be trivial with something like https://github.com/github/linguist

> Is there an ontology of research software we could use and ask people to select from?

I'm not aware of a research software ontology but there are subject taxonomies available e.g. this one: https://github.com/PLOS/plos-thesaurus

**Comment by labarba** [comment]
It would be useful to even add a single "Keywords" field in the submission form, with a text prompt saying something like _"Please enter keywords for the research domain to which this software contributes, using the PLOS subject taxonomy [link]."_

**Comment by nichtich** [comment]
related issue #677

**Comment by sdruskat** [comment]
Relating to this, here are two things:

- There seems to be some sort of taxonomy for reviews by research domain (group): the review tracks, see screenshot missing the "Misc" track). Couldn't these be reused for a preliminary classification taxonomy by research domain? Mapping to other taxonomies could be done by downstream applications for now.
![grafik](https://github.com/openjournals/joss/assets/3007126/8b4489a1-fe0e-48f4-b077-ceae14f0c02b)

- Looking at Crossref metadata for JOSS publications, I found that the "subject" that is recorded there is set to "General Earth and Planetary Sciences" + "General Environmental Science", even though publications are clearly in another domain, e.g., educational research (https://joss.theoj.org/papers/10.21105/joss.05742, first screenshot) or linguistics (https://joss.theoj.org/papers/10.21105/joss.04825, second screenshot). I'm not sure if these values are provided by JOSS or added by Crossref somehow per default, e.g., for the journal as a whole? From a publication mining perspective it'd be great if any subject metadata for the paper could be "put there" instead :).
![grafik](https://github.com/openjournals/joss/assets/3007126/93d07428-be39-457e-acb9-68fccb5738d0)
![grafik](https://github.com/openjournals/joss/assets/3007126/29910fc5-8b71-4028-a513-5b6d83ca8557)
